tensorflow_datasets/core/download/kaggle.py

```diff
@@ -132,6 +132,13 @@ def download_file(self, fname, output_dir):
     _run_kaggle_command(command, self._competition_name)
     return os.path.join(output_dir, fname)
 
+  def download_all_files(self):
```

Comment:
maybe we should make it a static function?
I'd also add the output_dir argument.

so that people can just call: KaggleDownloader.download_all("foo", "/tmp")?

Reply:
Okayyy, I done it :)